.github/CODEOWNERS

```diff
@@ -1,5 +1,5 @@
 # Global rule
-* @wilwade
+* @aramikm
```

Comment:
Removing Wil so he doesn't get added to all pr reviews. I'm adding myself since I don't know who wants to be added to all PR's. If you like to be in this list please let me know.